Sources/Rendering/Misc/RemoteView/index.js

```diff
@@ -225,6 +228,7 @@ export function extend(publicAPI, model, initialValues = {}) {
     'rpcMouseEvent',
     'rpcGestureEvent',
     'rpcWheelEvent',
+    'canvasElement',
```

Comment:
The `set` won't be that useful since canvas vs canvasElement are 2 different names.

Reply:
can you explain more ? in line 20 i am using the following

```js
model.canvasView = vtkCanvasView.newInstance({
    canvas: model.canvasElement,
});
```

Reply:
Yes but what I mean is by calling `obj.setCanvasElement(canvas)` won't update the `vtkCanvasView` canvas. So instead of putting __canvasElement__ inside a __setGet__ macro, I would be fine to only put it inside a __get__ macro.